orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/ingest_metadata_type.py

```diff
@@ -21,16 +21,16 @@ def ingest_metadata_type(context: AbstractComputeExecutionContext,
                          result: list[JobId],
                          scratch_dataset_name: HcaScratchDatasetName) -> Iterator[MetadataTypeFanoutResult]:
     """
-    For each file type, return a dynamic output over which we can later map
-    This saves us from hardcoding solids for each file type
+    For each metadata type, return a dynamic output over which we can later map
+    This saves us from hardcoding solids for each type
     """
-    for file_metadata_type in context.solid_config["metadata_types"]:
+    for metadata_type in context.solid_config["metadata_types"]:
```

Comment:
Nothing huge here, this is just changing the variable name to be more general/accurate.